app/components/concept_details_component.rb

```diff
@@ -69,11 +69,12 @@ def properties_set_by_keys(keys, concept_properties, exclude_keys = [])
     end
   end
 
-  def filter_properties(top_keys, bottom_keys, exclude_keys, concept_properties)
+  def filter_properties(top_keys, bottom_keys, exclude_keys, concept_properties) 
     all_keys = concept_properties&.keys || []
     top_set = properties_set_by_keys(top_keys, concept_properties, exclude_keys)
     bottom_set = properties_set_by_keys(bottom_keys, concept_properties, exclude_keys)
-    leftover = properties_set_by_keys(all_keys - top_keys - bottom_keys, concept_properties, exclude_keys)
+    leftover = properties_set_by_keys(all_keys, concept_properties, exclude_keys)
+    leftover = leftover.reject { |key, _| top_set.key?(key) }
```

Comment:
to complete `leftover = leftover.reject { |key, _| top_set.key?(key)  || bottom_set.key?(key) }`